lib/pacemaker/pcmk_injections.c

```diff
@@ -116,7 +116,7 @@ pcmk__inject_failcount(pcmk__output_t *out, cib_t *cib_conn, xmlNode *cib_node,
                             pcmk__xe_id(cib_node), NULL, NULL, NULL, name,
                             NULL, &output) == pcmk_rc_ok) {
 
-        if (crm_element_value_int(output, name, &failcount) != 0) {
+        if (crm_element_value_int(output, PCMK_XA_VALUE, &failcount) != 0) {
```

Comment:
Ouch, yep it looks like I was treating this like a regular XML attribute, when really it's an `<nvpair>`.

@kgaillot FYI, this logic assumes there's only one fail count attribute for a given resource, task, and interval. If there are multiple fail count attrs for a given resource+operation+interval -- for example, for a clone instance that was globally unique, failed, and then was made anonymous -- then this will fail to find any of them, because the XPath results will come in a wrapper object.

Reply:
I didn't know the latter case was possible, until yesterday when looking at some COMPAT code.